apps/backend/src/modules/groups/group.repository.ts

```diff
@@ -45,12 +46,14 @@ class GroupRepository implements Repository {
 		return Promise.resolve(null);
 	}
 
-	public async findAll(): Promise<GroupEntity[]> {
+	public async findAll(): Promise<GroupGetAllResponseDto> {
```

Comment:
Use entity type on repository level, either `Promise<{ items: GroupEntity[] }>` or `Promise<PaginationResponseDto<UserEntity>>` for cases with pagination